cns/restserver/metrics.go

```diff
@@ -64,11 +111,14 @@ func init() {
 		ipConfigStatusStateTransitionTime,
 		syncHostNCVersionCount,
 		syncHostNCVersionLatency,
+		allocatedIPCount,
```

Comment:
these need to be removed from the ipampool metrics so that they are not duplicated. i don't think that this will run due to the double registration of the same metrics

Reply:
Can we change the names of the metrics for now?
Once we confirm that the metrics are visible, we can revert the metric names and remove them from the IPAMPool Monitor.